aleph-client/src/connections.rs

```diff
@@ -183,6 +183,18 @@ pub trait SignedConnectionApi: ConnectionApi {
         status: TxStatus,
     ) -> anyhow::Result<TxInfo>;
 
+    /// Returns account id which signs this connection
+    fn account_id(&self) -> &AccountId;
+
+    /// Returns a [`KeyPair`] which signs this connection
+    fn signer(&self) -> &KeyPair;
+
+    /// Tries to convert [`SignedConnection`] as [`RootConnection`]
+    async fn try_as_root(&self) -> anyhow::Result<RootConnection>;
```

Comment:
actually lets add a default impl where we return an error like `not upgradable to root` or smth

Reply:
done